src/scales/interpolatedColorScale.ts

```diff
@@ -135,10 +108,10 @@ export module Scales {
      *     (linear/pow/log/sqrt). Default is "linear". @see {@link scaleType}
      *     for further options.
      */
-    constructor(colorRange: any = "reds", scaleType: string = "linear") {
```

Comment:
mfw `any`. Can we also take this opportunity to move the `constructor` to the beginning of the file, where devs might be expected to look for it?